deploy/Dockerfile

```diff
@@ -58,12 +58,12 @@ EOT
 # LEAVE THIS OUT if your application is NOT a proper Python package.
```

Comment:
We need to do this to get the `reformatters` module installed (so imports work the same as in dev)